tests/component-tests/uiOptions/js/UIOptionsTests.js

```diff
@@ -15,30 +15,150 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 (function ($) {
     "use strict";
 
-    $(document).ready(function () {
+    var customizedTocTemplate = "../../../../src/components/tableOfContents/html/TableOfContents.html";
+
+    /* Mixin grade for UIO test component */
+    fluid.defaults("fluid.uiOptions.testPrefsEditorBase", {
```

Comment:
Better to put all of the test grades, functions, and etc in the "fluid.tests" namespace. This should ensure that there are no accidental collisions with the actual component and functions under test, and keeps things semantically grouped together in the global object.